app/lib/backend_api_logic/routing_policy.rb

```diff
@@ -19,7 +23,7 @@ def initialize(service)
       def to_a
         rules = backend_api_configs.reordering { sift(:path_desc) }.each_with_object([]) do |config, rules|
```

Comment:
```suggestion
        rules = backend_api_configs.reordering { sift(:path_desc) }.each_with_object([]) do |config, collection|
```

Reply:
Bad rebase, thank you
There were some conflicts